Automattic/wp-calypso
# Site Monitoring: Add initial severity filter for error logs

Related to https://github.com/Automattic/dotcom-forge/issues/4449

## Proposed Changes

In this PR, I propose to add a Severity filter for PHP error logs and Request Type and Status filters for access logs.

![Screenshot 2023-11-28 at 10 43 21](https://github.com/Automattic/wp-calypso/assets/727413/a05305e1-3285-4bab-beeb-3bd83854620a)

![Screenshot 2023-11-28 at 10 43 31](https://github.com/Automattic/wp-calypso/assets/727413/b5f23750-ea75-4a4d-a658-7ddfc42fb965)

## Testing Instructions



1. Create a site with a Business plan
2. Navigate to Settings -> Hosting Configuration and activate hosting features
3. Navigate to Tools -> Site monitoring
4. Confirm that new filter fields in log tabs work fine, that only required parameters are preserved between tabs, check pagination, log auto-refresh and elements flow in different screen sizes

## Pre-merge Checklist
